spec/image_spec.rb

```diff
@@ -26,200 +24,248 @@
     end
   end
 
-  describe '#dpg_jp2_filename' do
-    context 'with a dpg tiff file' do
-      let(:input_path) { TEST_DPG_TIF_INPUT_FILE }
-
-      it 'indicates the default DPG jp2 filename' do
-        expect(assembly_image.dpg_jp2_filename).to eq TEST_DPG_TIF_INPUT_FILE.gsub('.tif', '.jp2').gsub('_00_', '_05_')
-      end
-    end
-
-    context 'with a file with no extension' do
-      let(:input_path) { '/path/to/a/file_with_no_00_extension' }
-
-      it 'indicates the default jp2 filename' do
-        expect(assembly_image.dpg_jp2_filename).to eq '/path/to/a/file_with_no_05_extension.jp2'
-      end
-    end
-  end
-
   describe '#create_jp2' do
     context 'when input path is blank' do
       let(:input_path) { '' }
 
       it 'does not run if no input file is passed in' do
-        expect { assembly_image.create_jp2 }.to raise_error
+        expect { assembly_image.create_jp2 }.to raise_error(RuntimeError)
       end
     end
 
     context 'when given an uncompressed compressed RGB tif with more than 4GB of image data', skip: 'This test will create a 4GB test image and a 4GB temporary image, so skipping by default.' do
+      let(:input_path) { File.join(TEST_INPUT_DIR, 'rgb.tif') }
+
       before do
-        generate_test_image(TEST_TIF_INPUT_FILE, compress: 'none', width: '37838', height: '37838')
+        generate_test_image(input_path, compress: 'none', width: '37838', height: '37838')
       end
 
       it 'creates the jp2 with a temp file' do
-        expect(File).to exist TEST_TIF_INPUT_FILE
-        expect(File).not_to exist TEST_JP2_OUTPUT_FILE
-        result = assembly_image.create_jp2(output: TEST_JP2_OUTPUT_FILE)
+        expect(File).to exist input_path
+        expect(File).not_to exist output_file
+        result = assembly_image.create_jp2(output: output_file)
         expect(assembly_image.tmp_path).not_to be_nil
         expect(result).to be_a_kind_of described_class
-        expect(result.path).to eq TEST_JP2_OUTPUT_FILE
-        expect(TEST_JP2_OUTPUT_FILE).to be_a_jp2
+        expect(result.path).to eq output_file
+        expect(output_file).to be_a_jp2
         expect(result.exif.colorspace).to eq 'sRGB'
-        jp2 = described_class.new(TEST_JP2_OUTPUT_FILE)
+        jp2 = described_class.new(output_file)
```

Comment:
I don't understand why this line is needed ... is this not the same as "result" above?  Also, maybe a better name is result_jp2 ?   Or is there a specific need to reload the file?  If so, can that be indicated with a comment?

Ditto for tests below.

Reply:
yes, don't think this is needed, probably a relic of a copy and paste job of porting old tests over to new style...will fix